docs/Tutorials/Orientation.md

```diff
@@ -0,0 +1,167 @@
+\cond NEVER
```

Comment:
Have you had a chance to build this with Doxygen? Some of the commands I don't think will work (e.g. `/note` I believe should be `\note`), and the markdown formatting looks somewhat odd: https://github.com/gsb76/spectre/blob/4afa2f550b41e9ff908bef7ef620affd71fcef97/docs/Tutorials/Orientation.md

You can compile the doxygen documentation on your local machine without the need to build the rest of the code. Let me know if you have issues or if Doxygen is rendering things as you expected and my memory is wrong.

Reply:
ping: https://github.com/gsb76/spectre/blob/1677d982141c5607401dc5825293ec50e7f95698/docs/Tutorials/Orientation.md

Reply:
It should look good now :+1: